site2/docs/security-token-admin.md

```diff
@@ -6,96 +6,87 @@ sidebar_label: Token authentication admin
 
 ## Token Authentication Overview
 
-Pulsar supports authenticating clients using security tokens that are based on
-[JSON Web Tokens](https://jwt.io/introduction/) ([RFC-7519](https://tools.ietf.org/html/rfc7519)).
+Pulsar supports authenticating clients using security tokens that are based on [JSON Web Tokens](https://jwt.io/introduction/) ([RFC-7519](https://tools.ietf.org/html/rfc7519)).
 
-Tokens are used to identify a Pulsar client and associate with some "principal" (or "role") which
-will be then granted permissions to do some actions (eg: publish or consume from a topic).
+You can use tokens to identify a Pulsar client and associate with some "principal" (or "role") that
+is permitted to do some actions (for example, publish to a topic or consume from a topic).
 
-A user will typically be given a token string by an administrator (or some automated service).
+A user typically gets a user a token string from the administrator (or some automated service).
 
-The compact representation of a signed JWT is a string that looks like:
+The compact representation of a signed JWT is a string that looks like as the follwing:
 
 ```
- eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJKb2UifQ.ipevRNuRP6HflG8cFKnmUPtypruRC4fb1DWtoLL62SY
- ```
+eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJKb2UifQ.ipevRNuRP6HflG8cFKnmUPtypruRC4fb1DWtoLL62SY
+```
+
+Application specifies the token when you are creating the client instance. An alternative is to pass a "token supplier" (a function that returns the token when the client library needs one).
```

Comment:
Check the first sentence, and pay attention to two points:
1. the word "specifies"
2. use present tense.